src/application.js

```diff
@@ -608,6 +608,11 @@ class XtkSchemaNode {
          */
         this.packageStatus = PACKAGE_STATUS[this.packageStatusString];
 
+        /**
+         * Returns a string which indicates the custom/extended entity, attribute belongs to.
```

Comment:
Can you be more accurate? It's not just a string that is returned here but the schemaId of the source schema where the attribute or element was originally defined. I mean I think that's what it is, can you also confirm that point?